src/openrpc/content.json

```diff
@@ -0,0 +1,561 @@
+                        "rating": "PG"
+                      },
+                      {
+                        "scheme": "CA-Movie",
+                        "rating": "G"
+                      }
+                    ],
+                    "waysToWatch": [
+                      {
+                        "identifiers": {
+                          "assetId": "123"
+                        },
+                        "expires": "2025-01-01T00:00:00.000Z",
+                        "entitled": true,
+                        "entitledExpires": "2025-01-01T00:00:00.000Z",
+                        "offeringType": "buy",
+                        "price": 2.99,
+                        "videoQuality": [
+                          "UHD"
+                        ],
+                        "audioProfile": [
+                          "dolbyAtmos"
+                        ],
+                        "audioLanguages": [
+                          "en"
+                        ],
+                        "closedCaptions": [
+                          "en"
+                        ],
+                        "subtitles": [
+                          "es"
+                        ],
+                        "audioDescriptions": [
+                          "en"
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        ]
+      },
+      {
+        "name": "requestUserInterest",
+        "tags": [
+            {
+                "name": "capabilities",
+                "x-uses": [
+                    "xrn:firebolt:capability:discovery:interest"
+                ]
+            }
+        ],
+        "params": [
+            {
+                "name": "type",
+                "schema": {
+                    "$ref": "https://meta.comcast.com/firebolt/discovery#/definitions/UserInterestType"
+                }
+            }
+        ],
+        "result": {
+            "name": "entity",
+            "schema": {
+                "$ref": "https://meta.comcast.com/firebolt/entertainment#/definitions/EntityInfo"
+            }
+        },
+        "examples": [
+            {
+                "name": "Default Example",
+                "params": [
+                    {
+                        "name": "type",
+                        "value": "interest"
+                    }
+                ],
+                "result": {
+                    "name": "entity",
+                    "value": {
+                        "identifiers": {
+                            "entityId": "xyz"
+                        },
+                        "entityType": "program",
+                        "programType": "movie",
+                        "title": "Interesting Movie Title"                        
+                    }
+                }
+            }
+        ]
+      },
+      {
+        "name": "onUserInterestedIn",
```

Comment:
Wondering if we can just drop the `In` and just be `onUserInterested`

`requestUserInterest`
`onUserInterestRequested`
`userInterest`
`onUserInterested`

Reply:
There's a use case where a voice assistant wants to tell the platform that the user is interested in something on the screen, e.g. "Hey Voice Assistant, this content is cool, save it to my list"

We need to send a generic `onUserInterest` intent to the device to initiate the flow in that case, so we need both names.